developer/src/vscode-plugin/package.json

```diff
@@ -31,30 +31,52 @@
         "properties": {},
         "when": ""
       }
+    ],
+    "customEditors": [
+      {
+        "viewType": "keyman.ldml",
+        "displayName": "Keyman LDML",
+        "selector": [
+          {
+            "filenamePattern": "*.xml"
+          }
+        ],
+        "priority": "default"
+      }
     ]
   },
   "scripts": {
-    "vscode:prepublish": "npm run compile",
-    "compile": "tsc -p ./",
+    "vscode:prepublish": "npm run compile && npm run build",
+    "compile": "tsc -p ./ && npm run build",
     "watch": "tsc -watch -p ./",
-    "pretest": "npm run compile && npm run lint",
+    "pretest": "npm run compile && npm run build && npm run lint",
     "lint": "eslint src",
     "test": "vscode-test",
-    "_prepare": "npm run compile"
+    "build": "parcel build --target=default --cache-dir=.parcel-cache --no-content-hash"
   },
   "devDependencies": {
     "@types/mocha": "^10.0.0",
     "@types/node": "20.4.1",
     "@types/vscode": "^1.93.0",
     "@vscode/test-cli": "^0.0.10",
-    "@vscode/test-electron": "^2.4.1"
+    "@vscode/test-electron": "^2.4.1",
+    "parcel": "^2.13.2"
   },
   "license": "MIT",
   "dependencies": {
     "@keymanapp/common-types": "file:../../../common/web/types",
     "@keymanapp/developer-utils": "file:../common/web/utils",
     "@keymanapp/kmc-kmn": "file:../kmc-kmn",
     "@keymanapp/kmc-ldml": "file:../kmc-ldml",
-    "@keymanapp/kmc-package": "file:../kmc-package"
+    "@keymanapp/kmc-package": "file:../kmc-package",
+    "process": "^0.11.10",
```

Comment:
needed "or else!" … or else, something attempts to install it into the root package.json